### ISSUE TITLE
RawTherapee doesn't start when in a different folder than /Applications/ on macOS

### ISSUE THREAD

**Issue by databu** [Beep6581/RawTherapee#7124]
**Short description**

I like to group my apps into sub-folders in `/Applications/`. But when I try to run RawTherapee from such a sub-folder (e.g. `/Applications/Sehen/` in my case), it doesn't start. It only starts when I put it directly in `/Applications/`.

**Steps to reproduce**

1. Create a sub-folder under `/Applications/`
2. Move (not copy) RawTherapee there.
3. Try to run RawTherapee from there.

**Expected behavior**
RawTherapee should start from any readable folder.

**Additional information**
- Version of RawTherapee 5.10 (also tried on 5.9)
- macOS Sonoma 14.3

Other useful information:
When I try to run it from the command line, I see:
```
$ /Applications/Sehen/RawTherapee.app/Contents/MacOS/rawtherapee 
dyld[58915]: Library not loaded: /Applications/RawTherapee.app/Contents/Frameworks/libomp.dylib
  Referenced from: <5807C8EA-F1C6-3FCF-84F1-FBC42D42C846> /Applications/Sehen/RawTherapee.app/Contents/MacOS/rawtherapee
  Reason: tried: '/usr/local/share/ImageMagick-6.7.5/lib//libomp.dylib' (no such file), '/Applications/RawTherapee.app/Contents/Frameworks/libomp.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/Applications/RawTherapee.app/Contents/Frameworks/libomp.dylib' (no such file), '/Applications/RawTherapee.app/Contents/Frameworks/libomp.dylib' (no such file), '/usr/lib/libomp.dylib' (no such file, not in dyld cache)
Abort trap: 6
```

The relevant line is:
```
'/Applications/RawTherapee.app/Contents/Frameworks/libomp.dylib' (no such file), 
``` 
which seems to indicate that this path is hardcoded into the app, rather than using a relative path from where the app is located?

**Comment by Benitoite** [comment]
Duplicate of https://github.com/Beep6581/RawTherapee/issues/6580

**Comment by Lawrence37** [comment]
TLDR: The hard-coded absolute path is used on purpose, to follow Apple's best-practice guidelines for security.

Closing as a duplicate.
